src/SUMMARY.md

```diff
@@ -222,7 +223,7 @@
       - [Calling Rust from C](android/interoperability/with-c/rust.md)
     - [With C++](android/interoperability/cpp.md))
     - [With Java](android/interoperability/java.md)
-- [Exercises](exercises/day-4/afternoon.md)
+- [Exercises](exercises/day-4/android.md)
```

Comment:
Please update [`book.toml`](https://github.com/google/comprehensive-rust/blob/main/book.toml#L44) accordingly to avoid a broken link.